test-cases/longevity/longevity-multidc-parallel-topology-schema-changes-12h.yaml

```diff
@@ -0,0 +1,30 @@
+test_duration: 800
+
+prepare_write_cmd:  "cassandra-stress write cl=QUORUM n=20971520 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=80 -pop seq=1..20971520 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"
```

Comment:
Don't you need to set here something like this:

'replication(strategy=NetworkTopologyStrategy,us-east1scylla_node_east=3,us-west1scylla_node_west=3)'

Reply:
fixed